### PR TITLE
LPS-138375 move jericho-html to shared-dependencies

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

This is for ticket https://issues.liferay.com/browse/LPS-138375.
I am trying to move jericho-html to shared-dependencies.
Thanks for checking

### CODE REVIEW

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
fixed code conflict

**Comment by Kvale1733** [comment]
@jeffwu0724 No regressions found.

File: modules/apps/shared-dependencies/shared-dependencies-jericho-html/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Bundle-Name: Liferay Shared Dependencies jericho-html
```
Comment (tinatian) [inline modules/apps/shared-dependencies/shared-dependencies-jericho-html/bnd.bnd:1]: @jeffwu0724 Please rename the bundle name to : Liferay Shared Dependencies **Jericho Html**

**Comment by tinatian** [comment]
@jeffwu0724 Please see my comment and resend.

Tina.

**Pull request closed without merging**